code/parse/sexp.cpp

```diff
@@ -12403,6 +12410,50 @@ void sexp_set_player_orders(int n)
 	}
 }
 
+void sexp_set_order_allowed_target(int n)
+{
+	bool allow_order;
+	std::set<size_t> orders;
+
+	auto ship_entry = eval_ship(n);
+	if (!ship_entry || !ship_entry->shipp) {
+		return;
+	}
+	auto shipp = ship_entry->shipp;
+
+	const std::set<size_t>& default_orders = ship_set_default_orders_against();
+	n = CDR(n);
+	allow_order = is_sexp_true(n);
+	n = CDR(n);
+	do {
+		for( size_t order : default_orders){
+			//Once we exceed the number of valid orders, break and warn
+			if (order >= Player_orders.size()) {
+				Warning(LOCATION, "Invalid order name %s found in sexp!", CTEXT(n));
+				break;
+			}
+			// OPF_AI_ORDER returns hud_name, so we must compare to that instead of parse_name
+			if (!stricmp(CTEXT(n), Player_orders[order].hud_name.c_str())) {
+				orders.insert(order);
+				break;
+			}
```

Comment:
A new line after this curly brace is not necessary, as there is another curly brace right after it, so there is no need to further highlight the if statement.

Reply:
All above feedback should be addressed now.